edXRegressionAutomated/Tests/LMS/pages/homepage.py

```diff
@@ -0,0 +1,12 @@
+from bok_choy.page_object import PageObject
+from . import BASE_URL
+
+class LMSHomePage(PageObject):
+    """
+    pages Home Page
+    """
+
+    url = BASE_URL
+
+    def is_browser_on_page(self):
+        return "free online courses " in self.browser.title.lower()
```

Comment:
the `self.browser.title` is one of the first things to render for a page, so this approach may result in flaky tests. You should look for something that is rendered on the page instead.